src/lib/finishPollMessage.ts

```diff
@@ -25,6 +25,15 @@ async function finishPoll(poll: IPoll, { persis }: { persis: IPersistence }) {
     return persis.updateByAssociation(association, poll);
 }
 
+async function getuserTimezone(read: IRead, data) {
+    const user = await read.getUserReader().getById(data.user.id);
+    const utcOffset = await user.utcOffset;
```

Comment:
```suggestion
    const utcOffset = await user.utcOffset;
```

You won't need an await here as accessing a prop within object isn't an async operation

Reply:
Done 😄👍